balarco/utils.py

```diff
@@ -8,6 +8,19 @@
 from rest_framework.authtoken.models import Token
 
 
+STATUS = (
+    (0, 'Pendiente'),
+    (1, 'Diseño'),
+    (2, 'Cuentas'),
+    (3, 'Validación'),
+    (4, 'Producción'),
+    (5, 'Por cobrar'),
+    (6, 'Por facturar'),
+    (7, 'Pagado'),
+    (8, 'Terminado')
```

Comment:
I know they mention both status (Pagado, Terminado) but are they really necessary?
I don't know if the 48 hours they wanted is for this state